MAINTAINERS.md

```diff
@@ -1,21 +1,25 @@
 # Maintainers
 
-MAINTAINERS:
+Code for this project is reviewed by any other member of the team before it is merged into the project. Team members include:
 
-- lucasmbrown-usds
-- aprilypchen
+## Project Leadership
```

Comment:
Seems like you're alphabetizing by last name, do you want people to add new folks to the bottom of the list or alphabetized? Maybe mention here in an inline comment for the future?